packages/planner/src/app/components_ngrx/planner-query/planner-query.component.less

```diff
@@ -25,15 +25,21 @@
     top: 10px;
   }
 
+  &__export-csv {
```

Comment:
Avoid using absolute layouts unless you have a very good reason to do so. Always find a way to do a flow layout. Either use flex here or responsive grid classes.